sdk/servicebus/perf-tests/service-bus-track-1/README.md

```diff
@@ -0,0 +1,10 @@
+### Guide
+
+1. Navigate to `sdk/servicebus/perf-tests/service-bus-track-1`.
+2. Do `rush update`.
+3. Run `npm run setup`.
+4. Create a service-bus namespace and populate the .env file with `SERVICEBUS_CONNECTION_STRING` variable.
+5. Run the tests as follows
+   - batch send
+     - `npm run perf-test:node -- BatchSendTest --warmup 2 --duration 7 --parallel 2`
+     - `npm run perf-test:node -- BatchSendTest --warmup 1 --duration 25 --iterations 2 --parallel 32 --messageBodySize 10240 --numberOfMessages 10`
```

Comment:
Same here, of course. Readmes could use some love ❤️

Reply:
Poured some love, let me know how they are